### PR TITLE
Doc: Zarr: clarify INTERLEAVE default value

### PR DESCRIPTION
**Pull request by rouault**
CC @mdsumner

### CODE REVIEW

**Comment by mdsumner** [comment]
you beat me to it 😀

File: doc/source/drivers/raster/zarr.rst [hunk 1]
```diff
@@ -505,6 +505,9 @@ The following options are creation options of the classic raster API only:
       When writing a multi-band dataset as a 3D Zarr array, whether the band
       dimension should be the first one/slowest varying one (BAND), or the
       last one/fastest varying one (INTERLEAVE)
```
Comment (mdsumner) [inline doc/source/drivers/raster/zarr.rst:507]: small addition: end of sentence should be (PIXEL)

Comment (rouault) [reply doc/source/drivers/raster/zarr.rst]: fixed

**Comment by mdsumner** [comment]
excellent, that's a very helpful doc addition

**Pull request merged**